lib/volume-parser.js

```diff
@@ -27,6 +28,10 @@ exports.parseVolume = (fieldTagV) => {
       volumes = extractVols(matches)
     }
   })
+  if (!volumes.length && fieldTagV.match(singletonRegex)) {
+    const vol = parseInt(fieldTagV.match(singletonRegex)[0], 10)
+    volumes = [[vol, vol]]
+  }
```

Comment:
Curious. Is there a reason this single number matching regex couldn't be included in the `regExp` array of patterns above? Is it just that it has only one match?

Reply:
i moved the logic to the extract vols function. it was a little tricky because theres only one capture group possible, whereas the other regular expressions generate multiple match arrays